.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: ci
```

Comment:
Oh this is so much nicer :) Thanks!

Reply:
More important, it should be notably faster! :-)